src/itensor.jl

```diff
@@ -2221,16 +2221,17 @@ function directsum(
   )
 end
 
-function directsum(A::ITensor, B::ITensor, I, J; tags)
+function directsum(A::ITensor, B::ITensor, I, J; kwargs...)
+  tags = get(kwargs, :tags, ["sum1", "sum2"])
   N = length(I)
   (N != length(J)) &&
     error("In directsum(::ITensor, ::ITensor, ...), must sum equal number of indices")
   IJ = Vector{Base.promote_eltype(I, J)}(undef, N)
   for n in 1:N
     In = I[n]
     Jn = J[n]
-    In = dir(A, In) != dir(In) ? dag(In) : In
-    Jn = dir(B, Jn) != dir(Jn) ? dag(Jn) : Jn
+    In = dir(A, In.first) != dir(In) ? dag(In) : In
+    Jn = dir(B, Jn.first) != dir(Jn) ? dag(Jn) : Jn
```

Comment:
I don't think this is correct, `I` and `J` should be lists of indices.

Reply:
Thanks. I wasn't totally clear on what types I and J were expected to have. I'll reevaluate how I tried to use `directsum` in ITensorTDVP and get a better understanding of it.